Optimize data filters in Console (SaaS) + business variable default (#9455)
camunda/camunda-docs: docs/components/hub/organization/manage-clusters/settings.md

Before you enable multi-tenancy checks, assign all users, groups, and roles that need access to their tenants and to the `<default>` tenant. Once checks are enforced, any principal not assigned to a tenant loses access to the resources scoped to that tenant.
:::

## Data filters

<!-- TODO: Confirm with Console team (camunda-cloud-management-apps#8885) which exact docs anchor the Console UI tooltip links to. Update the #data-filters anchor below if different. -->

You can configure data filters on a per-cluster basis to control which process definitions and variables the Optimize exporter processes.

:::note

> This setting applies to Camunda 8 SaaS. On Self-Managed, configure export filters using [Helm values or configuration properties](/self-managed/components/optimize/configuration/optimize-export-filtering.md).

Reviewer: Same comment as above: https://github.com/camunda/camunda-docs/pull/9455/changes#r3748790565

:::

Enable the **Enable data filters** toggle to activate filtering. When enabled, the Optimize exporter only processes data matching the configured filters.

Enter one pattern per line, or separate values with spaces, in any of the four fields:

- **Include process definitions**: process definitions to include, matched by exact `bpmnProcessId`. Leave empty to include all process definitions.
- **Exclude process definitions**: process definitions to exclude, matched by exact `bpmnProcessId`. Exclusion takes precedence over inclusion.
- **Include variable names**: variable names to include, matched by prefix. For example, entering `business_` includes all variables whose names start with `business_`. Leave empty to include all variables.
- **Exclude variable names**: variable names to exclude, matched by prefix. Exclusion takes precedence over inclusion.

:::note

> New SaaS clusters include a default `business_` variable include filter, which limits Optimize to variables whose names start with `business_`. Existing clusters show data filters disabled with a one-click opt-in — no automatic migration occurs. On Self-Managed, no default filter is applied; configure filters manually using [Helm values or configuration properties](/self-managed/components/optimize/configuration/optimize-export-filtering.md).

Reviewer: ```suggestion
New SaaS clusters include a default `business_` variable include filter, which limits Optimize to variables whose names start with `business_`. For existing clusters, data filters are disabled by default and can be enabled with one click. No automatic migration occurs. On Self-Managed, no default filter is applied; configure filters manually using [Helm values or configuration properties](/self-managed/components/optimize/configuration/optimize-export-filtering.md).
```
I would rephrase this sentence without the dash

:::

> :::warning

Reviewer: If possible, I would avoid placing two admonitions directly after one another

Before you enable multi-tenancy checks, assign all users, groups, and roles that need access to their tenants and to the `<default>` tenant. Once checks are enforced, any principal not assigned to a tenant loses access to the resources scoped to that tenant.
:::

## Data filters

> <!-- TODO: Confirm with Console team (camunda-cloud-management-apps#8885) which exact docs anchor the Console UI tooltip links to. Update the #data-filters anchor below if different. -->

Author: CC @ultraschuppi

Author: Will resolve in a separate PR once answered.

> <!-- TODO: Confirm whether data filters are available on trial plans or restricted to paid plans (Q6 - tied to Optimize re-enable on trial)? -->

Author: CC @shay-matasaro-camunda

Reviewer 2: @christinaausley filters are available for all plans .
 cc @felix-mueller
